src/frontend/js/pages/TeacherCourseDashboardLoader/index.tsx

```diff
@@ -0,0 +1,77 @@
+import { FormattedMessage, defineMessages } from 'react-intl';
+import { useParams } from 'react-router-dom';
+
+import { capitalize } from 'lodash-es';
+import { DashboardLayout } from 'widgets/Dashboard/components/DashboardLayout';
+import { TeacherCourseDashboardSidebar } from 'widgets/Dashboard/components/TeacherCourseDashboardSidebar';
+import { useCourse } from 'hooks/useCourses';
+import { Spinner } from 'components/Spinner';
+import { DashboardCard } from 'widgets/Dashboard/components/DashboardCard';
+import { Icon, IconTypeEnum } from 'components/Icon';
+import CourseRunList from './CourseRunList';
+
+const messages = defineMessages({
+  courses: {
+    defaultMessage: 'Your courses',
+    description: 'Filtered courses title',
+    id: 'components.TeacherCourseDashboardLoader.title.filteredCourses',
+  },
+  incoming: {
+    defaultMessage: 'Incoming',
+    description: 'Incoming courses title',
+    id: 'components.TeacherCourseDashboardLoader.title.incoming',
+  },
+  ongoing: {
+    defaultMessage: 'Ongoing',
+    description: 'Ongoing courses title',
+    id: 'components.TeacherCourseDashboardLoader.title.ongoing',
+  },
+  archived: {
+    defaultMessage: 'Archived',
+    description: 'Archived courses title',
+    id: 'components.TeacherCourseDashboardLoader.title.archived',
+  },
+  loading: {
+    defaultMessage: 'Loading course ...',
```

Comment:
```suggestion
    defaultMessage: 'Loading course…',
```

Reply:
`…` is the correct ponctuation but I don't know if it's easy to find this ponctuation on other OS than Mac OS and other keyboard than Azerty, so we can assume to use `...`.